src/instructlab/model/backends/backends.py

```diff
@@ -77,18 +85,26 @@ def __init__(
         model_family: str,
         model_path: pathlib.Path,
         chat_template: str,
-        api_base: str,
         host: str,
         port: int,
+        config: ServerConfig,
     ) -> None:
         self.model_family = model_family
         self.model_path = model_path
         self.chat_template = chat_template if chat_template else CHAT_TEMPLATE_AUTO
-        self.api_base = api_base
         self.host = host
         self.port = port
+        self.config = config
         self.resources: list[Closeable] = []
 
+        # Write logs to a file if log_file is provided
+        if self.config.log_file:
+            self.write_logs_to_file(logger)
+
+    def write_logs_to_file(self, a_logger: logging.Logger) -> None:
```

Comment:
This seems more like a setup than a write operation?

Reply:
Funny enough, I initially had `setup_logs_to_file`. I guess I'll use setup then. Thanks!